pipelines/e2e/scripts/longhorn-setup.sh

```diff
@@ -53,11 +53,7 @@ main(){
   setup_longhorn_ui_nodeport
   export_longhorn_ui_url
 
-  if [[ -n "${LONGHORN_TESTS_CUSTOM_IMAGE}" ]]; then
-    run_longhorn_e2e_test
-  else
-    run_longhorn_e2e_test_out_of_cluster
-  fi
+  run_longhorn_e2e_test_out_of_cluster
```

Comment:
nit: Do we want to change the name more specific, something like run_longhorn_e2e_test_dockerized ?

Reply:
I'll refine this in the following PR. Thank you!